test/rulesets/Base/arraymath.jl

```diff
@@ -15,12 +15,27 @@
         ⋆₂(a, b) = ⋆₂((a, b))  #matrix
         ⋆₂() = ⋆₂(())  # scalar
 
+        ⋆₃(a) = (⋆(n), ⋆(n, 1))
+
         @testset "Scalar-Array $dims" for dims in ((3,), (5,4), (2, 3, 4, 5))
             rrule_test(*, ⋆(dims), ⋆₂(), ⋆₂(dims))
             rrule_test(*, ⋆(dims), ⋆₂(dims), ⋆₂())
         end
 
         @testset "AbstractMatrix-AbstractMatrix" begin
+            # Matrix-Vector product
+            @testset "n=$n, m=$m" for n in (2, 3), m in (4, 5)
+                @testset "Array" begin
+                    rrule_test(*, ⋆(n), ⋆₂(n, m), ⋆₂(m))
```

Comment:
Is there a reason for not taking advantage of the infix operator in this case?
```suggestion
                    rrule_test(*, ⋆(n), n ⋆₂ m, ⋆₂(m))
```

Reply:
No reason at all, I've always thought that it makes no difference whether we use the `infix` operator or use it as a function call. As I can see using `btime` that they almost have the same time. Should we preferring one over the other, and if so why?

Reply:
Yes, it is purely syntax sugar.
Idea of it is to make it easier to see the sizes are lined up to be correct.
Which matters for the matrix matrix case below.
For other cases we can just he doing infix for consistency